packages/suite/src/components/suite/Metadata/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { resolveStaticPath } from '@suite-utils/nextjs';
+import { URLS } from '@suite-constants';
+import Head from 'next/head';
+import { useTranslation } from '@suite-hooks/useTranslation';
```

Comment:
ahh, type-check is failing and I suspect this is the reason (useTranslation is importing something from suite-types, which means whole file will get type-checked and suite-web-landing pkg is missing some type definitions)

If I am correct then just don't use useTranslation and instead directly use `react-intl`'s useIntl hook